webapp/src/components/EstateDetailPage/EstateDetail.css

```
@@ -121,6 +121,10 @@
   margin-top: 20px;
 }
 
+.EstateDetail .row .column.highlights{
```

```suggestion
.EstateDetail .row .column.highlights {
```

can i ask for a space here?